presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestPushAggregationThroughOuterJoin.java

```diff
@@ -46,18 +41,9 @@
     @Test
     public void testPushesAggregationThroughLeftJoin()
     {
-        FunctionCall averageFunctionCall = new FunctionCall(QualifiedName.of("avg"), ImmutableList.of(new SymbolReference("COL2")));
-        Signature avgSignature = new Signature(
-                "avg",
-                FunctionKind.AGGREGATE,
-                ImmutableList.of(),
-                ImmutableList.of(),
-                DOUBLE.getTypeSignature(),
-                ImmutableList.of(DOUBLE.getTypeSignature()),
-                false);
         new RuleTester().assertThat(new PushAggregationThroughOuterJoin())
-                .on(p ->
-                        p.aggregation(
+                .on(p -> p.aggregation(ab -> ab
```

Comment:
You could achieve that by having 
`PlanBuilder.onAggregation(BiConsumer<PlanBuilder, AggreagtionBuilder>` but this is very usecase specific and I do not believe worth pursuing.